### DISCUSSION TITLE
Precise cut when selecting a key frame

### DISCUSSION THREAD

**Discussion by Moreasan** [mifi/lossless-cut#1636] [Support] [answered]
I am aware that it is hard to do a precise cut without encoding, and I can use the Smart Cut (I do) which may produce better result.

But I thought that if I select a cut at the start of a key frame, the next frame won't be included. Am I mistaken? Because I am selecting a cut at key frame A, but in the resulted segment I am seeing that the next key frame B is also included.

I am using Linux version, and I am not seeing the option to select key frame cut or normal cur. Was it an old feature?

If you confirm that it is an issue, I will upload a video test.

**Comment by mifi** [comment]
click this icon first, then export

<img width="50" alt="Screenshot 2023-06-29 at 12 37 16" src="https://github.com/mifi/lossless-cut/assets/402547/7e9db9d8-a4e7-490e-bb7e-eac7012a3d9c">

**Comment by Moreasan** [comment]
I found out the issue. Here how I do the cut.

I type the start and end of a segment and then I navigate to next/previous key frames. I thought the start and the end are updated but it is not the case.

That's an unexpected behavior. Any reason for that?

**Reply by mifi** [reply]
i'm not sure what you mean by that. could you provide some screenshots and step-by-step instruction on what you did?

**Comment by Moreasan** [comment]
Here how I do:

1. I type the start and end of a segment (ex: 00:10:43.765 – 00:29:21.432)
2. I use buttons to go to next/previous frame.

I thought the start and end were being updated (for ex to: 00:10:43.264 – 00:29:21.976). But they weren't, and I need to click on the button to update the start and the end of the segment.

**Comment by mifi** [comment, marked as answer]
Right, this is how it's supposed to work. Clicking next/previous frame will only update the playhead cursor and preview. It will not update the start+end timestamps of the current segment.
